src/index.ts

```diff
@@ -1,22 +1,23 @@
 import * as engine from './engine';
-import { allAudienceDefinitions } from './audiences';
+import { audienceMap } from './audiences';
```

Comment:
Can we improve the naming here a little, I am not a fan adding data types to names e.g: `usersArray`